### ISSUE TITLE
NVDA doesn't read feeds or rss in Internet Explorer 9

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#1693]
**Reported by drein on 2011-07-27 09:10**
In Internet Explorer9 it appears NVDA is not able to read rss pages. For Example, try this:
http://groups.google.com/group/universal-access/feed/atom_v1_0_msgs.xml

**Comment by leonardder** [comment]
Could anyone provide info on how this works in IE11?

**Comment by bdorer** [comment]
I'm not able to reproduce with IE11 so wote for closing as worksforme.

**Comment by leonardder** [comment]
As this is a pretty old issue, I'm tempted to agree with closing this.
